### DISCUSSION TITLE
Delivery date

### DISCUSSION THREAD

**Discussion by judev-jbg** [saleweaver/python-amazon-sp-api#672] [FAQ]
Is it possible to obtain the range of delivery dates of the offers for an ASIN?
For example:

![image](https://user-images.githubusercontent.com/45266431/183592844-1c5f4317-70b5-4c36-8d8b-a5c62ac38601.png)

I am using the get_item_offers endpoint but the response only returns the minimum and maximum shipping time the product should be shipped, but not the date the product will be received.

![image](https://user-images.githubusercontent.com/45266431/183593237-b33508ba-9a3d-42f9-aa08-ef17304905cb.png)

Any ideas?

**Comment by cdargel** [comment]
How did you solve this?

**Comment by PH2LP** [comment]
Hello my friend. Ive been dealing with this for months and still not able to solve it. Were you able to solve it? Thank you
